### DISCUSSION TITLE
Should be apply a style guide and rename some packages?

### DISCUSSION THREAD

**Discussion by aolle** [WebGoat/WebGoat#1309] [Q&A] [answered]
Hi,

@nbaars , @zubcevic 

Checking the code, I saw that we have old packages with underscore in the name; for example, `path_traversal`.

Should we follow a style guide, like

https://google.github.io/styleguide/javaguide.html#s5.2-specific-identifier-names

and rename some packages?

What do you think?

Regards.

**Comment by nbaars** [comment, marked as answer]
Yes I agree, let's rename those packages.
